great_expectations/dataset/pandas_dataset.py

```diff
@@ -194,6 +194,62 @@ def inner_wrapper(self, column_A, column_B, mostly=None, ignore_row_if="both_val
         inner_wrapper.__doc__ = func.__doc__
         return inner_wrapper
 
+    @classmethod
+    def multicolumn_map_expectation(cls, func):
+        """
+        The multicolumn_map_expectation decorator handles boilerplate issues surrounding the common pattern of
+        evaluating truthiness of some condition on a per row basis across a set of columns.
+        """
+        if PY3:
+            argspec = inspect.getfullargspec(func)[0][1:]
+        else:
+            argspec = inspect.getargspec(func)[0][1:]
+
+        @cls.expectation(argspec)
+        @wraps(func)
+        def inner_wrapper(self, column_list, mostly=None, ignore_row_if="all_values_are_missing",
+                          result_format=None, *args, **kwargs):
+
+            if result_format is None:
+                result_format = self.default_expectation_args["result_format"]
+
+            test_df = self[column_list]
+
+            if ignore_row_if == "all_values_are_missing":
```

Comment:
All/Any/Never seems a good way to handle missingness. Will be interesting to see how it works out in practice.

Reply:
👍